gateway-ha/src/main/java/io/trino/gateway/ha/router/StatementUtils.java

```diff
@@ -191,13 +193,17 @@ private StatementUtils() {}
             .build().stream()
             .collect(toImmutableMap(StatementTypeInfo::getStatementType, identity()));
 
-    public static Optional<QueryType> getQueryType(Statement statement)
+    public static String getResourceGroupQueryType(Statement statement)
```

Comment:
Why did you rename this method?

Reply:
`TrinoQueryProperties` already uses the name `queryType` for the statement class name:
```
queryType = statement.getClass().getSimpleName();
resourceGroupQueryType = StatementUtils.getQueryType(statement).toString();
```
I changed the method name in this PR because I found myself confused while reading the code. I could put it in a separate commit if that is preferable. 

I would actually rather change `queryType` in  `TrinoQueryProperties` to `statementType`, but this would be a breaking change for those using `TrinoQueryProperties.getQueryType()` in their routing rules.

Reply:
I don't request a separate commit, but explaining such a context in  a PR description or leaving a review comment will be helpful.